src/org/javarosa/xpath/expr/XPathFuncExpr.java

```diff
@@ -194,9 +194,16 @@ public Object eval (DataInstance model, EvaluationContext evalContext) {
         } else if (name.equals("int")) { //non-standard
             assertArgsCount(name, args, 1);
             return toInt(argVals[0]);
-        } else if (name.equals("round")) { // non-standard Excel-style round(value,decimal place)
-            assertArgsCount(name, args, 2);
-            return round(toNumeric(argVals[0]), toNumeric(argVals[1]).intValue());
+        } else if (name.equals("round")) { // Proximate XPath 3.0 and Excel-style round(value,decimal place)
```

Comment:
Removed the assertion requiring two arguments. Sans a second argument, decimal places will be zero (0). Nevertheless does assert that there cannot be more than two (2) arguments. Handles numbers with comma (,) separated decimal place through string method replace.